### DISCUSSION TITLE
如何让vela-core忽略某个application的调谐操作

### DISCUSSION THREAD

**Discussion by wangqi811** [kubevela/kubevela#5902] [Q&A]
背景：已经创建成功application A，但是在日常使用中是使用不到它的，所以在创建或者更新完成application A后，会删除其创建的所有对应deployment，但是过application-re-sync-period周期后，所有deployment又会重新被拉起来，这个问题有办法解决吗？

版本：1.2

**Comment by Somefive** [comment]
加一个 controller.core.oam.dev/pause: "true" 这个 label 在这个 application 上

P.S. 这个是个 1.9.0-alpha.1 版本的 feature

Reference: https://github.com/kubevela/kubevela/pull/5732

1.2 版本是没有这个 feature

**Reply by wangqi811** [reply]
好的，版本一时还升不上来，我想想其他办法，感谢！
